playground/frontend/lib/main.dart

```diff
@@ -17,12 +17,14 @@
  */
 
 import 'package:flutter/material.dart';
+import 'package:flutter_issue_106664_workaround/flutter_issue_106664_workaround.dart';
```

Comment:
The package code: https://github.com/alexeyinkin/flutter-issue-106664-workaround/blob/main/lib/src/web.dart